src/redux/reducers/experimentSettings/initialState.js

```diff
@@ -1,24 +1,4 @@
-const initialPipelineState = {
-  startDate: null,
-  endDate: null,
-  status: null,
-  completedSteps: [],
-};
-
-const initialWorkerState = {
-  status: null,
-  started: false,
-  ready: null,
-  restartCount: 0,
-};
```

Comment:
Before the structure was like this: `backendStatus: {...state}` because we only had to worry about one experiment.

Now we have this: `backendStatus: {experiment1Id: {...state}, experiment2Id: {...state}}`, and when an experiment is in its initial state it just doesn't have an entry in backendStatus.

I did move parts of this to other parts where I was using it though, either way it needed to go from here because I am not storing the backend statuses in experimentSettings.

You can check the initial state in backendStatus.